### ISSUE TITLE
Update to 3.0.0

### ISSUE THREAD

**Issue by Aenoxic** [BlockHorizons/ItemRename#5]

**Comment by Wertzui123** [comment]
Yes Pleas update! It's a nice idea, but if its not working it isn't good.

**Comment by lukeeey** [comment]
This has been updated to PM 5

**Comment by Wertzui123** [comment]
lol, didn't expect to receive a notification from a 6 year old thread, but thanks for updating I guess 😅

**Comment by lukeeey** [comment]
> lol, didn't expect to receive a notification from a 6 year old thread, but thanks for updating I guess 😅 

I can't believe it's been so long tbh
